examples/gno.land/r/gov/dao/proxy.gno

```diff
@@ -0,0 +1,151 @@
+package dao
+
+import "std"
+
+// dao is the actual govDAO implementation, having all the needed business logic
+var dao DAO
+
+// AllowedDAOs contains realms that can be used to update the actual govDAO implementation,
+// and validate Proposals.
+// This is like that to be able to rollback using a previous govDAO implementation in case
+// the latest implementation has a breaking bug. After a test period, a proposal can be
+// executed to remove all previous govDAOs implementations and leave the last one.
+var AllowedDAOs []string
+
+// proposals contains all the proposals in history.
+var proposals *Proposals = NewProposals()
+
+// Render calls directly to Render's DAO implementation.
+// This allows to have this realm as the main entry point for everything.
+func Render(p string) string {
+	return dao.Render(p)
+}
+
+// MustCreateProposal is an utility method that does the same as CreateProposal,
+// but instead of erroing if something happens, it panics.
+func MustCreateProposal(r ProposalRequest) ProposalID {
+	pid, err := CreateProposal(r)
+	if err != nil {
+		panic(err.Error())
+	}
+
+	return pid
+}
+
+func ExecuteProposal(pid ProposalID) bool {
+	execute, err := dao.PreExecuteProposal(pid)
+	if err != nil {
+		panic(err.Error())
+	}
+
+	if !execute {
+		return false
+	}
```

Comment:
A panic could be used maybe to fail the TX when execution is not allowed, or instead maybe `PreExecuteProposal` could just return an error. Unless there is a use case for `PreExecuteProposal` to return false without error.

Reply:
executed is meant to check if the proposal was approved or discarded. I'll add some documentation for this.